### ISSUE TITLE
Missing interface methods (device.reverseTcpPort) in typings

### ISSUE THREAD

**Issue by dadski72** [wix/Detox#2803]
Hi,
I need to use the Detox `device.reverseTcpPort()` in typescript, but there is none.
So looks like the method is missing at
```
  node_modules/detox/index.d.ts
```

To be sure, I looked at `node_modules/detox/src/devices/Device.js`:

```js
async reverseTcpPort(port) {
  await this.deviceDriver.reverseTcpPort(this._deviceId, port);
}
```
and can see that it is implemented.

So I tried to use `@types/detox`.
The `reverseTcpPort()` method exists but does not have a port parameter.
```ts
 reverseTcpPort(): Promise<void>;
```

What can be the workarounds for this? 

Thanks

**Comment by noomorph** [comment]
@dadski72, thanks for the report. ETA is tomorrow.

**Comment by d4vidi** [comment]
@noomorph kudos for working on a fix. @dadski72 in any case, the method is there - you can use it with a `@ts-ignore`, for the time being.

**Comment by noomorph** [comment]
Resolved in [18.16.0](https://github.com/wix/Detox/releases/tag/18.16.0)
